### ISSUE TITLE
Unable to check labels and create training set

### ISSUE THREAD

**Issue by olliopti** [DeepLabCut/DeepLabCut#839]
Thanks for opening this issue, and thanks for using DeepLabCut (we hope you are enjoying it ;). Please fill out the template completely, including the full "traceback" and input code that you ran to hit this error.

**Your Operating system and DeepLabCut version**

Please state your operating system, env, and which version of DeepLabCut you are using.
Example: Ubuntu 16.04 LTS, with an Anaconda Env, & DeepLabCut1.x or 2.x.

Please complete the following information about your system:

OS: Debian GNU/Linux 10 (buster)
DeepLabCut Version: when using the code deeplabcut._version_ I get the error module 'deeplabcut' has no attribute '_version_'
Anaconda env used: Anaconda3

A clear and concise description of what the problem is.
I am unable to run the line of code to check labels and create the training set for my project. I have successfully set the config path as well as labeled the frames.
Please place code inside this:

deeplabcut.check_labels(path_config_file)
deeplabcut.create_training_dataset(path_config_file)

[Please copy/paste the full terminal output of the error here!!!]

Creating images with labels by Olivia.

---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-13-123499898180> in <module>
----> 1 deeplabcut.check_labels(path_config_file)

/usr/local/bin/anaconda3/envs/DLC/lib/python3.7/site-packages/deeplabcut/generate_training_dataset/trainingsetmanipulation.py in check_labels(config, Labels, scale, dpi, draw_skeleton, visualizeindividuals)
    506                 keypoint=Labels[0],
    507                 draw_skeleton=draw_skeleton,
--> 508                 color_by=color_by,
    509             )
    510         except FileNotFoundError:

/usr/local/bin/anaconda3/envs/DLC/lib/python3.7/site-packages/deeplabcut/utils/visualization.py in make_labeled_images_from_dataframe(df, cfg, destfolder, scale, dpi, keypoint, draw_skeleton, color_by)
    255     bones = []
    256     if draw_skeleton:
--> 257         for bp1, bp2 in cfg["skeleton"]:
    258             match1, match2 = [], []
    259             for j, bp in enumerate(bodyparts):

ValueError: too many values to unpack (expected 2)

/mnt/expanse/expansion/olivia/DLC-MWM-Olivia-2020-08-05/training-datasets/iteration-0/UnaugmentedDataSet_DLC-MWMAug5  already exists!

---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-11-b29a52be26d9> in <module>
----> 1 deeplabcut.create_training_dataset(path_config_file)

/usr/local/bin/anaconda3/envs/DLC/lib/python3.7/site-packages/deeplabcut/generate_training_dataset/trainingsetmanipulation.py in create_training_dataset(config, num_shuffles, Shuffles, windows2linux, userfeedback, trainIndices, testIndices, net_type, augmenter_type)
    899 
    900         Data = merge_annotateddatasets(
--> 901             cfg, Path(os.path.join(project_path, trainingsetfolder)), windows2linux
    902         )
    903         if Data is None:

/usr/local/bin/anaconda3/envs/DLC/lib/python3.7/site-packages/deeplabcut/generate_training_dataset/trainingsetmanipulation.py in merge_annotateddatasets(cfg, trainingsetfolder_full, windows2linux)
    591     videos = cfg["video_sets"].keys()
    592     for video in videos:
--> 593         _, filename, _ = _robust_path_split(video)
    594         if cfg.get("croppedtraining", False):
    595             filename += "_cropped"

/usr/local/bin/anaconda3/envs/DLC/lib/python3.7/site-packages/deeplabcut/generate_training_dataset/trainingsetmanipulation.py in _robust_path_split(path)
    575     sep = "\\" if "\\" in path else "/"
    576     parent, file = path.rsplit(sep, 1)
--> 577     filename, ext = file.split(".")
    578     return parent, filename, ext
    579 

ValueError: too many values to unpack (expected 2)

**How to Reproduce the problem**
1. Run cells with code mentioned above. 

**Screenshots**

**Additional context**

**Comment by MMathisLab** [comment]
Please fill out the template as requested, otherwise it is impossible to help you with your question. :)

**Comment by jeylau** [comment]
@orutler, the second error you report has been fixed in DLC2.2b7. As for the first, it looks like there is something wrong with the way the skeleton is defined in the config. Using the `SkeletonBuilder` may help you here 😃

**Comment by olliopti** [comment]
Hello @MMathisLab  and @jeylau, thank you for your responses. Although I am new to coding and Deeplabcut, I am enjoying it immensely, thank you for creating this wonderful program! I updated the template. In the meantime I have successfully gotten to the train network step. My video files had periods within their names and my config file included spaces within the body part descriptions. I started a new project and made sure to leave periods and spaces out.
